addons/default/visiosoft/advs-module/resources/views/list/gallery.twig

```diff
@@ -36,6 +44,19 @@
             </div>
         {% endfor %}
     </div>
+    <script>
+        $('.ads-img').first().css('display', 'block');
```

Comment:
Do you need this?

Reply:
deleted. also added show() and hide() functions instead of css.(....)